doc/check-options.md

```diff
@@ -147,6 +147,7 @@ All checks allow these global options:
 | `boldValue`                                                 | `700`   | The minimum CSS `font-weight` value that designates bold text                                               |
 | `boldTextPt`                                                | `14`    | The minimum CSS `font-size` pt value that designates bold text as being large                               |
 | `largeTextPt`                                               | `18`    | The minimum CSS `font-size` pt value that designates text as being large                                    |
+| `shadowOutlineEmMax`                                        | `0.1`   | Maximum thickness of `text-shadow` in `em` at which it can server as an outline                             |
```

Comment:
I had to figure out why thickness meant based on the code. Rather than guessing it would be best to make it very clear what the option looks at and does.

```suggestion
| `shadowOutlineEmMax`                                        | `0.1`   | The maximum `blur-radius` value (in ems) of the CSS `text-shadow` property. `blur-radius` values greater than this value will be treated as a background color rather than an outline color.                             |
```